### PR TITLE
Feature/create login page

### PR DESCRIPTION
**Pull request by GGrify**

### CODE REVIEW

**Comment by Quipex** [comment]
Run eslint --fix against all files you've commited

File: frontend/src/screens/Login/components/LoginForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+const LoginForm: React.FC<ILoginFormProps> = (
+  { children }
+) => {
+
+  const toggleShowPassword = () => {
+    const input = document.getElementById('password');
+    const type = input.getAttribute('type');
+    const svgButton = document.getElementById('viewButton');
```
Comment (Quipex) [inline frontend/src/screens/Login/components/LoginForm/index.tsx:21]: Try not to use dom api directly when you're working with React

File: frontend/src/screens/Login/components/LoginForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          '              <path d="M0.999909 1.70707L3.0274 3.73455C1.68459 4.75008 0.606588 6.14168 0.0267382 7.8385C-0.00891273 7.94348 -0.00891273 8.05676 0.0267382 8.16174C1.22011 11.654 4.42425 14.0002 8 14.0002C9.48642 14.0002 10.8903 13.5634 12.1288 12.836L14.2931 15.0002L15.0001 14.2932L1.70692 1L0.99988 1.70704L0.999909 1.70707ZM8.00003 13.0002C4.90723 13.0002 2.12738 10.9978 1.03019 8.00015C1.57134 6.52153 2.54381 5.31884 3.74111 4.44827L5.55992 6.26707C5.21038 6.75762 4.99998 7.35326 4.99998 8.00012C4.99998 9.65443 6.34569 11.0002 8.00003 11.0002C8.64688 11.0002 9.24253 10.7898 9.73308 10.4402L11.397 12.1042C10.3643 12.6638 9.21405 13.0002 8.00003 13.0002ZM6.28333 6.99051L9.00969 9.71688C8.71221 9.89255 8.36986 10.0002 8.00003 10.0002C6.89697 10.0002 6.00001 9.10321 6.00001 8.00018C6.00001 7.63034 6.10766 7.288 6.28333 6.99051Z" fill="#919193" fill-opacity="0.5"/>\n' +
+          '            </svg>';
```
Comment (Quipex) [inline frontend/src/screens/Login/components/LoginForm/index.tsx:38]: Extract the custom input with hiding password to a separate reusable component

**Review by Quipex — CHANGES_REQUESTED** [review]
When you're creating new reusable components like buttons, modals, icons, etc. Tend to make them separate components so that others can use them

**Review by AlenaGavriluk — COMMENTED** [review]
There is no need to use {} with attributes values e.g. placeholder={"Enter your email"}
RegistrationForm/index.tsx and LoginForm/index.tsx  should be formatted with tabs.
<label htmlFor={"name"}>Name</label>
        <input id={"name"} type="text" placeholder={"Enter your name"} required/>
 <label htmlFor={"surname"}>Surname</label>

File: frontend/src/screens/Login/components/HideSvg/index.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+export interface IHideSvgProps extends IState, IActions {
+}
+
+interface IState {
+}
+
```
Comment (kopot-o-d) [inline frontend/src/screens/Login/components/HideSvg/index.tsx:9]: u don't need interfaces and mapState.. mapDispatch... for dull components, u can just remove all unnecessary constructions

File: frontend/src/screens/Login/components/UnhideSvg/index.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      <path d="M9.98642 7.86534L10.8882 8.76715C10.9538 8.52082 11.0001 8.26685 11.0001 8.00005C11.0001 6.34574 9.65435 5 8.00001 5C7.73321 5 7.47924 5.04622 7.23291 5.11183L8.13472 6.01364C9.12862 6.08126 9.9188 6.87144 9.98642 7.86534Z" fill="#919193" fill-opacity="0.5"/>
+      <path d="M0.999909 1.70707L3.0274 3.73455C1.68459 4.75008 0.606588 6.14168 0.0267382 7.8385C-0.00891273 7.94348 -0.00891273 8.05676 0.0267382 8.16174C1.22011 11.654 4.42425 14.0002 8 14.0002C9.48642 14.0002 10.8903 13.5634 12.1288 12.836L14.2931 15.0002L15.0001 14.2932L1.70692 1L0.99988 1.70704L0.999909 1.70707ZM8.00003 13.0002C4.90723 13.0002 2.12738 10.9978 1.03019 8.00015C1.57134 6.52153 2.54381 5.31884 3.74111 4.44827L5.55992 6.26707C5.21038 6.75762 4.99998 7.35326 4.99998 8.00012C4.99998 9.65443 6.34569 11.0002 8.00003 11.0002C8.64688 11.0002 9.24253 10.7898 9.73308 10.4402L11.397 12.1042C10.3643 12.6638 9.21405 13.0002 8.00003 13.0002ZM6.28333 6.99051L9.00969 9.71688C8.71221 9.89255 8.36986 10.0002 8.00003 10.0002C6.89697 10.0002 6.00001 9.10321 6.00001 8.00018C6.00001 7.63034 6.10766 7.288 6.28333 6.99051Z" fill="#919193" fill-opacity="0.5"/>
+    </svg>
+  );
+};
+
```
Comment (kopot-o-d) [inline frontend/src/screens/Login/components/UnhideSvg/index.tsx:25]: u don't need interfaces and mapState.. mapDispatch... for dull components, u can just remove all unnecessary constructions

**Review by kopot-o-d — COMMENTED** [review]
move ALL SVGs to separate components

**Comment by GGrify** [comment]
> There is no need to use {} with attributes values e.g. placeholder={"Enter your email"}
> RegistrationForm/index.tsx and LoginForm/index.tsx should be formatted with tabs.
> <label htmlFor={"name"}>Name
> <input id={"name"} type="text" placeholder={"Enter your name"} required/>
> <label htmlFor={"surname"}>Surname

**Comment by GGrify** [comment]
> There is no need to use {} with attributes values e.g. placeholder={"Enter your email"}
> RegistrationForm/index.tsx and LoginForm/index.tsx should be formatted with tabs.
> <label htmlFor={"name"}>Name
> <input id={"name"} type="text" placeholder={"Enter your name"} required/>
> <label htmlFor={"surname"}>Surname

I think it's prettier when all the attributes are in brackets

**Review by kopot-o-d — APPROVED** [review]

**Pull request merged**